### DISCUSSION TITLE
How to enable case-insensitive completion in startship?

### DISCUSSION THREAD

**Discussion by MikeChongCan** [starship/starship#3814] [Q&A] [answered]
I did a quick search but cannot find it.

[This issue](https://github.com/starship/starship/issues/2821) mentioned this environment variable it does not help:

```
# Set to this to use case-sensitive completion
CASE_SENSITIVE="true"
```

**Comment by davidkna** [comment, marked as answer]
Starship is only a prompt and not a shell configuration framework like oh-my-zsh and thus does not contain options for this.

Case insensitive completions are shell-specific, what shell are you using?

**Reply by MikeChongCan** [reply]
Thanks for the explanation! I am using `zsh` but disabled `oh-my-zsh` after installing starship.

**Reply by davidkna** [reply]
Try `unsetopt case_glob`. You might also want to look at setting some history options if you haven't already, I believe `zsh` doesn't store shell history by default.

**Reply by MikeChongCan** [reply]
Will do. Thanks a lot!
